### ISSUE TITLE
Super Table display 

### ISSUE THREAD

**Issue by kasgithub8** [poirazis/bb-component-SuperTable#46]

**Comment by poirazis** [comment]
hey @kasgithub8 
v 2 is in Alpha, so if this a production usage, please revert to the latest 1.x
v 2 is also not backwards compatible, and in general it really a completely different component from scratch.

**Comment by kasgithub8** [comment]
Hey @poirazis,

yeah, I reverted Super Table and Super Table Column back to 2.0.9.
Now there's some small issues like not being able to sort in the columns but that's fine.

Is there a way for me to use a value from Column A in Column B? For example: I have title & URL and want to add a Link component that has the title for the text and the URL for the link. That would be huge, because every time I add the ColumnA.Value in the Column B, it's empty.

**Comment by gswec2005** [comment]
how can i use 2.0.9 version of plugin?
its not showing data

**Comment by kasgithub8** [comment]
> how can i use 2.0.9 version of plugin?
@gswec2005 You follow these links:
1. https://github.com/poirazis/bb-component-SuperTable/releases/tag/v2.0.9-alpha
2. https://github.com/poirazis/bb-component-SuperTableColumn/releases/tag/v2.0.9-alpha

on those release sites you can download the ...2.0.9-alpha.tar.gz and add those in the plugin tab on budibase per file upload rather than GitHub.
![image](https://github.com/poirazis/bb-component-SuperTable/assets/95284278/e91385ef-40f9-4fdd-b358-af53326a5e3f)

**Comment by Aleksey1555** [comment]
Hello.
I also revert SuperTable and SuperTableColumn to version 2.0.9.
And have this error:
![image](https://github.com/poirazis/bb-component-SuperTable/assets/32514002/efac0b4c-728e-4ea9-9c62-28c726144cc9)
Please Help!!!

**Comment by kasgithub8** [comment]
@Aleksey1555 I don't know how the plugins work and if they keep the settings even when you downgrade, but does this happen also happen with a completely new table and new columns?

**Comment by Aleksey1555** [comment]
yes, it happen's. I just have created new application with new simple Table (id,Name) and , in Design ,added Data Provider and SuperTable. And the problem still present.

**Comment by Aleksey1555** [comment]
Ploblem solved. I installed last version and the problem is solved!
